build-tools/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04
+
+# Add i386 arch and update apt cache
+# RUN dpkg --add-architecture i386 && apt-get update
+RUN apt-get update
+
+# Install WebRTC base dependencies
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    apt-utils \
+    ca-certificates \
+    ccache \
+    curl \
+    git \
+    python \
+    python-setuptools \
+    python3 \
+    sudo \
+    wget \
+    xz-utils
+
+# Install WebRTC for Linux compile dependencies
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    libxml2 \
+    pkg-config
+
+# Add user
+ARG USERNAME=docker-webrtc
+ARG UID=1000
+ARG GID=1000
```

Comment:
It's great that this is parametrized, but right now the parameter is never overridden with your actual UID. This way, you'll run into permission issues if your local user's UID is not 1000.

To avoid this problem, you can pass the actual UID/GID values to `docker build` with the `--build-arg` parameter (`id -u` and `id -g`).